docs/reference/run.mdx

```diff
@@ -115,6 +116,12 @@ The current supported fields to filter in kernel are:
  * `-c string`, `--containername string`, show only data from containers with
    that name. Notice that this container name is the one set by the runtime, and
    it may be different from the one set by Kubernetes depending on the runtime.
+* `--k8s-containername string`, show only data from containers with the name
```

Comment:
```suggestion
* `-c string`, `--runtime-containername string`, show only data from containers with
that name. Notice that this container name is the one set by the runtime, and
it may be different from the one set by Kubernetes depending on the runtime.
* `--k8s-containername string`, show only data from containers with the name
```